napari/_qt/_tests/test_exception_handler.py

```diff
@@ -29,15 +32,50 @@ def test_keyboard_interupt_handler(qtbot, capsys):
     assert capsys.readouterr().err == "Closed by KeyboardInterrupt\n"
 
 
-def test_exception_handler_gui(qtbot, make_napari_viewer):
-    """Test exception handler can create a NapariNotification"""
-    viewer = make_napari_viewer()
-    handler = ExceptionHandler(gui_exceptions=True)
-    with qtbot.waitSignal(handler.error, timeout=1000):
-        handler.handle(ValueError, ValueError("whoops"), None)
-    assert handler.message in viewer.window.qt_viewer.canvas.native.children()
-    handler.message.toggle_expansion()
-    assert handler.message.property('expanded') is True
-    handler.message.toggle_expansion()
-    assert handler.message.property('expanded') is False
-    handler.message.close()
+# def test_exception_handler_gui(qtbot, make_napari_viewer):
+#     """Test exception handler can create a NapariQtNotification"""
+#     viewer = make_napari_viewer()
+#     handler = ExceptionHandler(gui_exceptions=True)
+#     with qtbot.waitSignal(handler.error, timeout=1000):
+#         handler.handle(ValueError, ValueError("whoops"), None)
+#     assert handler.message in viewer.window.qt_viewer.canvas.native.children()
+#     handler.message.toggle_expansion()
+#     assert handler.message.property('expanded') is True
+#     handler.message.toggle_expansion()
+#     assert handler.message.property('expanded') is False
+#     handler.message.close()
```

Comment:
This one is removed as the Exception Handler is not connected anymore;  I'll likely delete that.